src/nvim/tag.c

```diff
@@ -2407,8 +2407,10 @@ jumpto_tag (
   /* If it was a CTRL-W CTRL-] command split window now.  For ":tab tag"
    * open a new tab page. */
   if (postponed_split || cmdmod.tab != 0) {
-    win_split(postponed_split > 0 ? postponed_split : 0,
-        postponed_split_flags);
+    if (win_split(postponed_split > 0 ? postponed_split : 0,
+                  postponed_split_flags) == FAIL) {
+      WLOG(_("Error splitting window"));
```

Comment:
Inspecting the implementation of win_split() shows that it does `EMSG` for all failure cases, so we don't need to log it here.

Easiest thing to do here is ignore the return value (otherwise we need to do some cleanup and might require some refactoring.  jumpto_tag() can deal with a failed split, so it's no big deal.

    (void)win_split(...);